base_product_mass_addition/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# © 2019 Today Akretion
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base Product Mass Addition",
+    "version": "13.0.1.0.0",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "license": "AGPL-3",
+    "category": "Product",
+    "depends": ["stock", "onchange_helper"],
```

Comment:
IMHO, you should depends on product only as people maybe want to use it having only sale installed for instance (and don't want to install stock one)

Reply:
Thanks @rousseldenis for your review. We will have separated PR to fix in 13.0 then port it to 14.0.
@dalonsod, I updated this PR again so that you can keep super(...) logic so please ignore my previous comment. Sorry for the confusion. And, please note about rousseldenis's comment.
> And on the same idea, adds the qty_available related fields in your example 'stock_picking_quick'